src/components/ComeBackFeedback/ComeBackFeedback.css

```diff
@@ -0,0 +1,56 @@
+.comeback-feedback {
+    width: 100vw;
+    border-top: 1px solid #E5E5E5;
+    text-align: center;
+    margin: 150px auto 0px auto;
+}
+
+img {
+    max-width: 450px;
+}
+
+strong {
+    font-size: 23pt;
```

Comment:
in general, it's good to avoid using pixels for font sizes. instead, we like to use a metric called `rem`, which allows font sizes to change based on the font size that a user sets for their browser! for example, if someone can't read small text very well and sets their browser's font size to be "very large", then rem will respond to that and enlarge text on the page, while pixels will not change it. For medium sized font (the default), you can use the conversion 16px = 1rem.